editor/docks/scene_tree_dock.cpp

```diff
@@ -2898,6 +2898,18 @@ void SceneTreeDock::_update_script_button() {
 	button_create_script->set_visible(can_create_script);
 	button_detach_script->set_visible(can_detach_script);
 	button_extend_script->set_visible(can_extend_script);
+
+	update_script_button_queued = false;
+}
+
+void SceneTreeDock::_queue_update_script_button() {
+	if (update_script_button_queued) {
+		return;
+	}
+
+	update_script_button_queued = true;
+
+	callable_mp(this, &SceneTreeDock::_update_script_button).call_deferred();
```

Comment:
```suggestion
	}
	update_script_button_queued = true;
	callable_mp(this, &SceneTreeDock::_update_script_button).call_deferred();
```
Unnecessary empty lines.

Reply:
Done.